de/cheatsheet-deep-learning.md

```diff
@@ -1,162 +1,162 @@
 
-&#10230;
+&#10230; Im Ergebnis wird das Gewicht wie folgt aktualisiert:
 
 <br>
 
 **14. Updating weights ― In a neural network, weights are updated as follows:**
 
-&#10230;
+&#10230; Das Aktualisieren der Gewichtungen - In neuronalen Netzen werden die Gewichtungen wie folgt aktualisiert:
 
 <br>
 
 **15. Step 1: Take a batch of training data.**
 
-&#10230;
+&#10230; Schritt 1: Nimm einen Stapel von Lerndaten.
 
 <br>
 
 **16. Step 2: Perform forward propagation to obtain the corresponding loss.**
 
-&#10230;
+&#10230; Schritt 2: Führe Vorwärtsausbreitung durch um den entsprechenden Verlust zu erhalten.
 
 <br>
 
 **17. Step 3: Backpropagate the loss to get the gradients.**
 
-&#10230;
+&#10230; Schritt 3: Fehlerrückführe den Verlust um die Gradienten zu erhalten.
 
 <br>
 
 **18. Step 4: Use the gradients to update the weights of the network.**
 
-&#10230;
+&#10230; Schritt 4: Verwende die Gradienten um die Gewichtungen im Netz zu aktualisieren.
 
 <br>
 
 **19. Dropout ― Dropout is a technique meant at preventing overfitting the training data by dropping out units in a neural network. In practice, neurons are either dropped with probability p or kept with probability 1−p**
 
-&#10230;
+&#10230; Aussetzen - Aussetzen ist eine Technik um eine Überanpassung der Lerndaten zu verhindern bei der Einheiten in einem neuronalen Netz ausfallen. In der Praxis werden Neuronen entweder mit Wahrscheinlichkeit p ausgesetzt oder mit Wahrscheinlichkeit 1-p behalten.
 
 <br>
 
 **20. Convolutional Neural Networks**
 
-&#10230;
+&#10230; Faltende neuronale Netzwerke
 
 <br>
 
 **21. Convolutional layer requirement ― By noting W the input volume size, F the size of the convolutional layer neurons, P the amount of zero padding, then the number of neurons N that fit in a given volume is such that:**
 
-&#10230;
+&#10230; Vorraussetzung für eine faltende Schicht - Sei W das Eingangsvolumen, f die Größe der Neuronen der faltenden Schicht, P die Anzahl der aufgefüllten Nullen, dann ist die Anzahl der Neuronen N die in ein gegebenes Volumen passen:  
 
 <br>
 
 **22. Batch normalization ― It is a step of hyperparameter γ,β that normalizes the batch {xi}. By noting μB,σ2B the mean and variance of that we want to correct to the batch, it is done as follows:**
 
-&#10230;
+&#10230; Bündelnormalisierung - Ein Schritt des Hyperparameters γ,β welcher das Bündel {xi} normalisiert. Es seien μB der Mittelwert und σ2B die Varianz dessen um was das Bündel korrigiert werden soll, dann gilt folgendes:
 
 <br>
 
 **23. It is usually done after a fully connected/convolutional layer and before a non-linearity layer and aims at allowing higher learning rates and reducing the strong dependence on initialization.**
 
-&#10230;
+&#10230; Geschieht üblicherweise nach einer vollständig verbundenen/faltenden Schicht und vor einer nicht-linearen Schicht und bezweckt eine höhere Lernrate und eine Reduzierung der starken Abhängigkeit von der Initialisierung.
```

Comment:
~~Geschieht~~ **Wird** üblicherweise nach einer ~~vollständig verbundenen~~ **kompletten**/faltenden und vor einer nicht-linearen Schicht **durchgeführt** und bezweckt ~~eine höhere~~ die **Erhöhung** der Lernrate und eine Reduzierung der starken Abhängigkeit vom initialen Wert der Lernrate.

Reply:
Is "komplett" the same as / known as / used as "vollständig verbunden" in the literature?